clients/src/main/java/org/apache/kafka/clients/admin/internals/DescribeConsumerGroupsHandler.java

```diff
@@ -354,11 +356,18 @@ private void handleError(
             case GROUP_ID_NOT_FOUND:
                 if (isConsumerGroupResponse) {
                     log.debug("`{}` request for group id {} failed because the group is not " +
-                        "a new consumer group. Will retry with `DescribeGroups` API.", apiName, groupId.idValue);
+                        "a new consumer group. Will retry with `DescribeGroups` API. {}",
```

Comment:
nit: Do you think that printing the error message is necessary here and below?

Reply:
I think a debug log line is appropriate. There's a chance that diagnostics in this area might show something not quite right as users are migrating their clusters to the new protocol.